docker/couchdb/monitor.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/bash
 
 # start couchdb exporter
-nohup couchdb-prometheus-exporter -telemetry.address=":9984" 2>&1 1>& couchdb_exporter.log < /dev/null &
+COUCH_CONFIG=/data/srv/current/auth/couchdb/couchdb_config.ini
+# test if file is not zero size
+if [ -s "${COUCH_CONFIG}" ]; then
```

Comment:
I do not know if you'll find it useful, but I suggest to further separate this if block into two if blocks. First, should check if $COUCH_CONFIG file exist and if not make a copy from secrets. The second should check if file is empty or not. This separation should avoid copying secrets over and over if we'll re-turn monitor.sh again and again (when you restart within a pod). But this suggestion is not necessary per-se and I'll approve the PR.

Reply:
This would add code duplication, given that the file is already copied in the Dockerfile.